gov/governance/config.gno

```diff
@@ -0,0 +1,110 @@
+package governance
+
+import (
+	"std"
+
+	"gno.land/p/demo/ufmt"
+
+	"gno.land/r/gnoswap/v2/common"
+	"gno.land/r/gnoswap/v2/consts"
+)
+
+var (
+	config         Config
+	configVersions = make(map[uint64]Config)
+)
+
+func init() {
+	// value for testing case from @notJoon
+	config = Config{
+		VotingStartDelay:              uint64(100),
+		VotingPeriod:                  uint64(1000),
+		VotingWeightSmoothingDuration: uint64(100),
+		Quorum:                        uint64(500),
+		ProposalCreationThreshold:     uint64(100),
```

Comment:
What does this do? can we set a minimum holding of GNS of a user to create a proposal with this?

Reply:
purpose of each configuration value are described here
https://docs.gnoswap.io/core-concepts/governance#scope-of-proposals-5

`ProposalCreationThreshold` <- this is required amount of token for user to create proposal.